File: Core/Entities/Answer.cs
namespace Core.Entities;

public class Answer : BaseEntity
{
    public string Text { get; set; } = null!;
    
    public User? Author { get; set; }
    public int AuthorId { get; set; }
    
    public DateTime DatePosted { get; set; } = DateTime.Now;
    
    public DateTime? LastModifiedDate { get; set; }
    
    public string? PictureUrl { get; set; }
    
    public Question? Question { get; set; }
    public int QuestionId { get; set; }

    public virtual List<Vote>? Votes { get; set; } = new List<Vote>();

    public int Score { get; set; } = 0;
}

File: StackOverflowFE/src/app/shared/models/answer.ts
export interface IAnswer {
    id: number;
    authorUsername: string;
    text: string;
    datePosted: Date;
    lastModifiedDate: Date;
    pictureUrl: string;
    score: number;
}

export interface IPostAnswer {
    id: number;
    text: string;
    questionId: number;
    pictureUrl: string;
}
